xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -37,19 +42,15 @@ constexpr const char* userFontPath = "special://home/media/Fonts/";
 // Directory where Kodi bundled fonts (default ones like Arial or Teletext) are located
 constexpr const char* systemFontPath = "special://xbmc/media/Fonts/";
 
-std::string GetDefaultFontPath(std::string& font)
+std::string GetSystemFontPath(std::string font)
```

Comment:
Maybe adjust comment on line 42, to refer the font families or the actual filenames

Reply:
Any reason for removing the ref? Could be const ref here

Reply:
i have removed ref due to the call with FONT::FONT_DEFAULT_FILENAME
but i think i can change std::string with const char*